### ISSUE TITLE
Max UI scale in local coop

### ISSUE THREAD

**Issue by raine** [thespbgamer/ZoomLevel#15]
### Describe the bug

Hey, I'm trying to use this mod to solve the problem that in 4K resolution, the UI scale is way too small in local coop.

So far haven't had much luck. Following questions:

1. Even with Max Zoom & UI Scale setting to 250%, I still can't change the UI scale in the settings to over 150%. Is this normal?
2. In console, the `reset_UI` command seems to work for setting larger than vanilla UI scale, but it only sets it for one player in local coop. Is there a way around this?

This is more of a question than bug report, but the the Github issue template is quite strict.

Thanks in advance.

### To Reproduce

–

### Version

Latest

### Settings

Defaults, except or max zoom & UI scale set to maximum possible value.

**Comment by thespbgamer** [comment]
I'll have to test this to give you extra information, but it might take a while to give you feedback, when possible, will look into this

**Comment by thespbgamer** [comment]
@raine 
From what I tested, it works fine with controller and keyboard, the diference between 200% and 250% is not THAT noticeable in co-op, can you confirm that the options really are in the 250% in the options, if not, it might be an 4K issue that I can't test because I don't have a 4K screen, might be something to check later.

TLDR: Confirm the options because in my end works fine, if possible, try to send prints of the issue.

**Comment by raine** [comment]
Thanks for testing. Here's what it looks like.

Zoom or UI Max Level set to 250%:
![20230307083147_1](https://user-images.githubusercontent.com/11027/223341639-9f60b2ce-079a-4acb-96a0-736ea2f9ad8e.jpg)

UI Scale in settings doesn't go higher than 150%:
![20230307083026_1](https://user-images.githubusercontent.com/11027/223341682-208954b2-5351-42bd-84f7-507707681593.jpg)

As you can see the UI is quite small at 150% scale.

The reason why I don't just play at 1080p fullscreen is that for some reason in exclusive fullscreen the game audio starts to crackle and cut off periodically. Took me a while to figure out that borderless fullscreen is the solution, so currently I have to set Windows resolution to 1080p just for Stardew Valley and play in borderless fullscreen. However, I'd rather keep Windows resolution at 4k always.

**Comment by thespbgamer** [comment]
Ohh, i'm sorry, maybe I didn't clarify, should explain better, the limits only change if you use console or the keybinds, the limits in the menus don't change (to not bore you with details, it's super difficult to change those menus)

**Comment by raine** [comment]
As far as I can tell, the keybind that increases UI scale and zoom level (should there be separate keybinds for these?) doesn't go past 150% UI scale either 😔.

**Comment by raine** [comment]
I can confirm that it doesn't work. The keybind only changes zoom level.

**Comment by thespbgamer** [comment]
By holding ``Left Trigger & Right Trigger`` and using the ``left stick`` to decrease the UI Scale or ``right stick`` to increases the UI Scale.
Assuming you are using the default controls and this is not working it's odd,  I'm gonna look into it but from what I tested this week, everything is working on an 1080p screen. I don't have a 4K display or funds to get one since this is a side project I do for fun, but if anything comes up, I'll release an update for it.

**Comment by raine** [comment]
Damn, I totally missed the part about triggers in the README. In the in game settings it wasn't that clear.

Thanks for clarifying, it seems to work in 1080p at least. I'll do some more testing at 4k later.

**Comment by thespbgamer** [comment]
No problem, now I have something to clarify later, not a high priority, but nice to know I have to clarify that, when possible, please give me the feedback if the problem still persists

**Comment by thespbgamer** [comment]
Gonna assume that the error was solved.
